shipyard_blueprints/hubspot/shipyard_hubspot/hubspot.py

```diff
@@ -186,36 +177,40 @@ def import_data(self, filename, import_data: dict):
             return response
 
         else:
-            handle_request_errors(response)
+            HubspotUtility.handle_request_errors(response)
 
     def import_contact_data(
-        self,
-        import_name: str,
-        filename: str,
-        import_operations: str,
-        object_type: str = "contacts",
```

Comment:
Removed the default values for object types which were causing false positive tests